src/arch/aarch64/bootinfo.rs

```diff
@@ -7,11 +7,13 @@ pub struct BootInfo {
 	pub magic_number: u32,
 	pub version: u32,
 	pub base: u64,
+	pub ram_start: u64,
 	pub limit: u64,
 	pub image_size: u64,
 	pub tls_start: u64,
 	pub tls_filesz: u64,
 	pub tls_memsz: u64,
+	pub tls_align: u64,
```

Comment:
Would we want to keep `BootInfo`'s structure as similar as possible across the architectures? For x86, I added it to the end to disrupt as little as possible. If we shuffle around the positions for x86 as well, we must pay attention to succeed the CI dance.

Reply:
Do we need that both architectures have the BootInfo? Inlong-term the arch64 port should uses device trees. I am not sure if we do it also on x86.

Reply:
Ah, okay. If we plan to move away from `BootInfo` on aarch64, it's not as important to keep them synchronized. :+1: